src/support/CommandRunner.ts

```diff
@@ -0,0 +1,71 @@
+import { CommandLineArgs } from '../support/commandLineBuilder';
```

Comment:
Is this from the docker extension? There is likely some things that are unnecessary to keep here. I generally advise that when adapting from other code you should only keep the relevant code unless there is a good reason to keep the whole.